.azure/pipelines/ci.yml

```diff
@@ -19,6 +38,8 @@ jobs:
   parameters:
     # Ensures the alignment of branch name and deployment params
     buildArgs: '/warnaserror:BUILD1001'
+    ${{ if or(eq(variables['Build.Reason'], 'Schedule'), and(eq(variables['System.TeamProject'], 'internal'), eq(variables['Build.Reason'], 'Manual'), eq(parameters.runCodeQL3000, 'true'))) }}:
```

Comment:
Note the schedule will run in public too. This condition doesn't handle that correctly. We want to run `CodeQL3000` internally only